server/api/services/posts.service.ts

```diff
@@ -24,6 +24,15 @@ export class PostsService {
     const doc = (await post.save()) as IPostModel;
     return doc;
   }
+
+  //this function uses the post_id and returns an updated postModel
+  async updatePostById(data: IPostModel, post_id: number): Promise<IPostModel> {
+    l.info(`update users data ${data}`);
```

Comment:
NIT: Switch "users" to "post's" to be more accurate

Reply:
Good catch, I'll change that now